### PR TITLE
seaweedfs: init at 2.07

### PR DESCRIPTION
**Pull request by raboof**
###### Motivation for this change

Submitted on behalf of kvtb who chose not to create a GitHub account yet.

https://discourse.nixos.org/t/casual-nixpkgs-contributions/9607/7

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/seaweedfs/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/networking/seaweedfs/default.nix:22]: I am not 100% sure here but I think this defaults to the platforms go supports and should only be changed if not all of those are supported.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>seaweedfs</li>
  </ul>
</details>

File: pkgs/applications/networking/seaweedfs/default.nix
Comment (raboof) [reply pkgs/applications/networking/seaweedfs/default.nix]: looks like you're right (https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L244), removed

File: pkgs/applications/networking/seaweedfs/default.nix
Comment (danieldk) [inline pkgs/applications/networking/seaweedfs/default.nix:10]:
```suggestion
    rev = version;
```

File: pkgs/applications/networking/seaweedfs/default.nix
Comment (danieldk) [inline pkgs/applications/networking/seaweedfs/default.nix:18]: New derivations need maintainers. If you are going to proxy future updates to this derivation, you could add yourself as a maintainer :wink: .

**Review by danieldk — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/seaweedfs/default.nix
Comment (raboof) [reply pkgs/applications/networking/seaweedfs/default.nix:18]: Getting somewhat mixed messages here (https://discourse.nixos.org/t/casual-nixpkgs-contributions/9607/2) - but happy to add myself as maintainer.

**Review by danieldk — APPROVED** [review]
LGTM. Thanks for making the changes. I made a small change to the `maintainers` attr (since `lib` was already in scope).

Result of `nixpkgs-review pr 102939` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>seaweedfs</li>
  </ul>
</details>

I'll merge after ofborg is also happy.

**Comment by danieldk** [comment]
Weird, GitHub is in an inconsistent state. This PR is merged, but it's still saying open for me.

**Pull request closed without merging**
